MinimedKit/PumpEvents/PumpEvent.swift

```diff
@@ -21,3 +21,16 @@ public protocol PumpEvent : DictionaryRepresentable {
     }
     
 }
+
+public extension PumpEvent {
+    public func canBeDelayedAppend() -> Bool {
```

Comment:
We should pick one of the cleaner paths

1. Make PumpModel available as part of the PumpEvent interface (or a new construct)
2. Separate PumpModel from PumpEvent

Preference?